restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -195,10 +196,10 @@
 	"externalServiceStepChooseTargetLabel": "Seleccione módulo de destino",
 	
 	"faxStepFaxLabel": "Contenido fax",
-	"faxStepFaxLabelPopover": "Escriba el contenido fax como texto. de Type the fax content as text. Utilice las variables con el signo de dollar como '$variablename'",
+	"faxStepFaxLabelPopover": "Escriba el contenido fax como texto. Utilice las variables con el signo de dollar como '$variablename'",
 	"faxStepToLabel": "Para",
 	"faxStepFromLabel": "De",
-	"faxStepContinueTooltip": "Seleccione el módulo para continuar después de que el fax se ha agregado a la cola. Las variables 'FaxStatus' y 'FaxSid' estarán disponibles. Déjelo en blanco para continuar con el siguiente paso.",
+	"faxStepContinueTooltip": "Seleccione el módulo para continuar después de que el fax se haya agregado a la cola. Las variables 'FaxStatus' y 'FaxSid' estarán disponibles. Déjelo en blanco para continuar con el siguiente paso.",
 	"faxStepContinueLabel": "Continuar con",
 	"faxStepChooseTargetLabel": "Elija un destino",
 	"faxStepStatusTooltip": "Escriba un URL para recibir una notificación acerca del estatus final de envío de fax. Una solicitud POST se hará con los parámetros: 'FaxStatus' y 'FaxSid'",

```

Comment:
Additionally, change this in the corresponding line (224):
"emailStepStatusTooltip": "Escriba la URL para recibir una notificación acerca del estado final de envío de fax. Una solicitud POST se hará con los parámetros: 'FaxStatus' y 'FaxSid'",